k8s/ingress_old.yml

```
@@ -0,0 +1,16 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: online-shop-ingress
+  namespace: online-shop-ns  
+spec:
+  rules:
+  - http:
+      paths:
+        - pathType: Prefix
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)